tests/lightning_tensor/test_measurements_class.py

```diff
@@ -126,3 +126,26 @@ def test_not_supported_shadowmp_shot_measurements(self):
 
             with pytest.raises(TypeError):
                 m.measure_tensor_network(tape)
+
+    @pytest.mark.parametrize("n_qubits", range(4, 14, 2))
+    @pytest.mark.parametrize("n_targets", list(range(1, 4)) + list(range(4, 14, 2)))
+    def test_probs_many_wires(self, n_qubits, n_targets, tol):
```

Comment:
This test. Would not take too long? :thinking:

Reply:
It's longer than sv backends but should be acceptable.